docs/pages/management/guides/docker.mdx

```diff
@@ -81,12 +80,12 @@ $ mkdir -p ~/teleport/config ~/teleport/data
 # This container will write the config and immediately exit--this is expected.
 $ docker run --hostname localhost --rm \
   --entrypoint=/usr/local/bin/teleport \
-  ${TELEPORT_DOCKER_IMAGE} configure > ~/teleport/config/teleport.yaml
+  ${TELEPORT_DOCKER_IMAGE} configure --roles=proxy,auth > ~/teleport/config/teleport.yaml
```

Comment:
Do we need to use the home directory here? This might be unexpected for a user who wants to create their own project directory for this demo

Reply:
This is how we've had it for quite a while.  This lets them spin it up in a user account dir vs requiring `sudo` checks an things.  I would leave it as is.